lib/plan_executor/app.rb

```diff
@@ -27,19 +28,25 @@ def puppetdb_client
       end
     end
 
-    def initialize(modulepath, executor = nil)
-      @schema = JSON.parse(File.read(File.join(__dir__, 'schemas', 'run_plan.json')))
-      @worker = Concurrent::SingleThreadExecutor.new
+    def initialize(modulepath, conn_opts)
+      conn_opts['User-Agent'] = "Bolt/#{Bolt::VERSION}"
+      @client = OrchestratorClient.new(conn_opts, true)
```

Comment:
This should be `false`, since we definitely _don't_ want to load the default config files.